_i18n/pt_BR/factors/dependencies.md

```diff
@@ -0,0 +1,55 @@
+**Idealmente, um aplicativo iOS-factor nunca depende da existência implícita de pacotes a nível de sistema**. Ela declara todas dependências, completamente e exatamente através de um manifesto de declaração de dependências. Isso inclui as versões exatas do [Xcode](https://developer.apple.com/xcode), [CocoaPods](https://cocoapods.org) e [fastlane](https://fastlane.tools).
```

Comment:
I think here `Ela declara` is related to `um aplicativo`, so I would change it to `Ele declara`